Source/DafnyRuntime/DafnyRuntime.py

```diff
@@ -161,18 +160,14 @@ def __le__(self, other):
         return len(self) <= len(other) and self == other[:len(self)]
 
 class Array:
-    class Box(list):
-        def __dafnystr__(self) -> str:
-            return '[' + ', '.join(map(string_of, self)) + ']'
-
     def __init__(self, initValue, *dims):
-        self.arr = initValue
+        def create_structure(initValue, *dims):
+            return [initValue if len(dims) <= 1 else create_structure(initValue, *dims[1:]) for _ in range(dims[0])]
         self.dims = list(dims)
-        for i in reversed(self.dims):
-            self.arr = Array.Box([copy.copy(self.arr) for _ in range(i)])
+        self.arr = create_structure(initValue, *dims)
 
     def __dafnystr__(self) -> str:
-        return '[' + ', '.join(map(string_of, self.arr)) + ']'
+        return f'array{self.dims}'
```

Comment:
Is this more consistent across backends or less?

Reply:
More. But, as the following program reveals:

``` dafny
method Main() {
  var a := new int[13];
  var m := new int[6, 28];
  print a, " ", m, "\n";
}
```

we still have ways to go for full consistency. For some targets (Go and half of JavaScript), we still print elements of the arrays. I'd rather we fix this issue in a separate PR.